serverless.yml

```diff
@@ -1,228 +1,233 @@
-  #  s3:
-  #    host: 0.0.0.0
-  #    port: 8001
-  #    directory: /tmp
-  #  capacities:
-  #    - table: UsersTable  # DynamoDB Resource
-  #      read:
-  #        minimum: 5       # Minimum read capacity
-  #        maximum: 100     # Maximum read capacity
-  #        usage: 0.75      # Targeted usage percentage
-  #      write:
-  #        minimum: 5       # Minimum write capacity
-  #        maximum: 100     # Maximum write capacity
-  #        usage: 0.75      # Targeted usage percentage
-  #    - table: JobsTable   # DynamoDB Resource
-  #      index:             # List or single index name
-  #        - ProducerIdGlobalIndex
-  #        - CrewIdGlobalIndex
-  #      read:
-  #        minimum: 4       # Minimum read capacity
-  #        maximum: 200     # Maximum read capacity
-  #        usage: 0.75      # Targeted usage percentage
-  #      write:
-  #        minimum: 2       # Minimum write capacity
-  #        maximum: 200     # Maximum write capacity
-  #        usage: 0.5       # Targeted usage percentage
-  prune:
-    automatic: true
-    number: 3
-  envFiles: # YAML files used to create .env file
-    - env.yml
-  envEncryptionKeyId: # KMS Key used for encrypting values
-    local: ${file(kms_key.yml):local} # Key used for local-stage
-    dev: ${file(kms_key.yml):dev} # Key used for development-stage
-    test: ${file(kms_key.yml):test} # Key used for test-stage
-    prod: ${file(kms_key.yml):prod} # Key used for prod-stage
-  #  serverless-offline-sqs:
-  #    autoCreate: true # create queue if not exists, it doesn't work for FIFO SQS
-  #    apiVersion: '2012-11-05'
-  #    endpoint: http://0.0.0.0:9324
-  #    region: ${file(env.yml):${opt:stage, self:provider.stage}.REGION}
-  #    accessKeyId: root
-  #    secretAccessKey: root
-  #    skipCacheInvalidation: false
-
-plugins: # order matters!
-  - '@redtea/serverless-env-generator'
-  - serverless-dir-config-plugin
-  - serverless-webpack
-  #  - serverless-offline-sqs
-  - serverless-offline
-  #  - serverless-offline-sns
-  #  - serverless-s3-local
-  - serverless-prune-plugin
+service: template-sls
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  environment:
+    STAGE: ${opt:stage, self:provider.stage}
+    ICONIK_TOKENS_TABLE_NAME: ${self:custom.tablesNames.IconikTokensTable.${opt:stage, self:provider.stage}}
+    USERS_TABLE_NAME: ${self:custom.tablesNames.UsersTable.${opt:stage, self:provider.stage}}
+    JOBS_TABLE_NAME: ${self:custom.tablesNames.JobsTable.${opt:stage, self:provider.stage}}
+  region: ${file(env.yml):${opt:stage, self:provider.stage}.REGION}
+  profile: ${file(env.yml):${opt:stage, self:provider.stage}.PROFILE}
+  tags:
+    client: ${file(env.yml):${opt:stage, self:provider.stage}.CLIENT}
+  iamRoleStatements:
+    - Effect: Allow
+      Action:
+        - cloudformation:DescribeStacks
+      Resource:
+        - arn:aws:cloudformation:*:*:stack/${file(env.yml):${opt:stage, self:provider.stage}.CLOUDFORMATION_STACK_NAME}/*

```

Comment:
Probably, you can use `StackName` pseudo parameter instead env var. Check the docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html#cfn-pseudo-param-stackname

Reply:
@emahota-flo We don't need this permission because we don't use the Cloud Formation service for getting API URL anymore